### DISCUSSION TITLE
Döngü Konuları

### DISCUSSION THREAD

**Discussion by volkantash** [turkce-yazilim-konati/yazilimca#57] [Ölçünler]
Selamun aleyküm.

Döngüler söz konusu olunca bir iki kullanım öne çıkıyor. Benim de şu ara düşündüğüm bir sözdizim düzeni var. Şöyle göstereyim.

```
i = 0 olup i < 1000 iken i++:
   (döngü içinde işlenecek işlemler)
```

Şimdi "**olup**" yerine "**kılıp**" mı desek, "**edip**" mi desek diye düşünüyorum. Başka bir sözcük dahi kullanabiliriz. Söz konusu yapı bence çok güzel duruyor. Sizin görüşleriniz nelerdir?

**Comment by turkmenistanli** [comment]
Aleyküm selam, ben kıl- eyleminden yanayım. Çünkü et- eylemi daha çok yardımcı eylem olarak işletilir
